example/main.dart

```diff
@@ -153,92 +145,97 @@ class _FlutterScreenState extends State<FlutterScreen> {
                             },
                           ),
                         ),
+
                         const SizedBox(
                           height: 12,
                         ),
                         ElevatedButton(
                           style: ElevatedButton.styleFrom(primary: Colors.red),
-                          onPressed: () {
-                            leave();
+                          onPressed: () async {
+                            // Step 6: Call leaveConference()
+                            await leaveConference();
                           },
-                          child: isLeaving
-                              ? const Text('Leaving...')
-                              : const Text("Leave conference"),
+                          child: isJoining
```

Comment:
this looks like mistake. isJoining but text = 'Leave the conference'

Reply:
We will have to fix this in beta 3.